_pages/ringcentral.html

```diff
@@ -0,0 +1,27 @@
+---
+layout: special-offer
+title: Export call recordings from RingCentral
+description: Break your calls free from RingCentral
+sitemap: false
+discount: true
+summary: 
+image_hero: /assets/images/sources/ringcentral.png
+image_body: /assets/images/logos/logo-evolveip.png
+summary_title: Export your calls to another service
+summary_body: |
+    Vendor lock-in hurts everyone. TransferVox makes it easy to automatically and continuously export your calls to another service.
+challange_title: Export Options&colon;
+challange_body: |
+  We support a myriad of export endpoints. Whether you need to get your calls from RingCentral to an SFTP server, or you want to migrate your call recordings to encrypted AWS S3 storage for long-term and cost effective archiving — we are prepared to make it simple. We also support Gong.io, Chorus.ai, and other third-party tools.
+strategy_title: Why use TransferVox?
+strategy_body: |
+  We specialize solely in securely and reliably transferring call recordings from a variety of vendors. Our systems are robust and single-focused, allowing us to minimize downtime and ensure that each and every recording is transferred as expected. Even if an endpoint goes down, our system keeps track of the outage and will automatically catch back up once the endpoint is restored.
+strategy_col1_title: Encrypted and Secured
+strategy_col1_body: |
+  We only utilize fully encrypted communications to transfer your recordings. This means everything is secured in transit and at rest. 
+strategy_col2_title: Reliability
+strategy_col2_body: |
+  Maintain your compliance by ensuring that you have a secondary storage for your call recordings. 
+conclusion: |
+  The use of the Evolve IP name and logo do not constitute an endorsement or certification by Evolve IP of TransferVox. We are an independent company that provides services to Evolve IP customers.
```

Comment:
@judsonlmoore is there a way to set a variable above (like `name: RingCentral` and have it replaced inline for other front matter?) 

as in, instead of having to edit each of these, I can just create generic templates and change one variable for each of the originator services.

Reply:
@edireson  I've simplified this with a layout "evolveip" for just that use-case, and another one "landing-page" where you just have to fill out minimal frontmatter and the texts and discounts get filled in.